apps/trip_plan/lib/trip_plan/transfer.ex

```diff
@@ -38,8 +38,16 @@ defmodule TripPlan.Transfer do
 
   def is_subway_transfer?(_), do: false
 
-  @doc "Takes a set of legs and returns true if there might be a transfer between the legs, based on the lists in @single_ride_transfers and @multi_ride_transfers. Exception: no transfers from bus route to same bus route."
+  @doc """
+  Takes a set of legs and returns true if there might be a transfer between the legs, based on the lists in @single_ride_transfers and @multi_ride_transfers.
+
+  Exceptions:
+  - no transfers from bus route to same bus route
+  - no transfers from a shuttle to any other mode
+  """
   @spec is_maybe_transfer?([Leg.t()]) :: boolean
+  def is_maybe_transfer?([%Leg{mode: %TransitDetail{route_id: "Shuttle-" <> _}} | _]), do: false
```

Comment:
Maybe one day we will have a better way to do this than parsing the name of the route.. 🙃

Reply:
Yeah, I totally agree. I didn't see anything better to use, but if there's something I missed or something in the future that would definitely be better.